### PR TITLE
Restructure/expand documentation

### PR DESCRIPTION
**Pull request by oliversalzburg**
The documentation has been restructured to reflect the structure of the UI. This should make it easier to find the documentation for a specific UI section.

Please review this and see if I made any mistakes in the newly added information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,69 +21,81 @@ You can also permanently install Kitten Scientists with a userscript manager.
...
+- **Hunt**: Automatically send your kittens hunting as you approach the catpower limit. This automation has a configurable trigger. The default value is `0.98`, which means the kittens will hunt when your catpower is at 98% of your catpower limit.
+- **Trade Blackcoin**: Automatically trade blackcoin with the leviathans at low prices and sells near the peak.
+- **Build Embassies (Beta)**: _pending documentation_
+- **Explore (deprecated)**: _relates to a feature no longer present in current versions of Kittens Game_
 
-Automatically observes astronomical events.
+### Filters
 
-All KS log output can be filtered.
+In this section, you can set which types of messages you want to see in the game log.
```
Comment (cameroncondry) [inline README.md]: > In this section, you can

I'm not sure how, but might want to remove this redundancy in the document. That starting line is repeated in almost every section.

File: README.md [hunk 2]
```diff
@@ -19,69 +21,81 @@ You can also permanently install Kitten Scientists with a userscript manager.
...
 
-### Game Log
+- **Observe Astro Events**: Automatically observe astronomical events as they happen.
+- **Hold Festivals**: Automatically hold festivals as soon as the previous one ends and enough resources are available.
+- **Auto Praise**: Automatically praises the sun as your faith approaches the resource limit.
+- **Force Ships to 243**: Ensures that 243 ships are constructed as soon as possible, ignoring other resource constraints. This is useful because after 243 ships, trades are guaranteed to be successful 100% of the time.
+- **Feed Leviathans**: Automatically feed the leviathans race necrocorns to ensure they stay longer for possible trading.
+- **Hunt**: Automatically send your kittens hunting as you approach the catpower limit. This automation has a configurable trigger. The default value is `0.98`, which means the kittens will hunt when your catpower is at 98% of your catpower limit.
+- **Trade Blackcoin**: Automatically trade blackcoin with the leviathans at low prices and sells near the peak.
+- **Build Embassies (Beta)**: _pending documentation_
```
Comment (Wymrite) [inline README.md]: Build Embassies is essentially the same as any of the building sections, just specifically for embassies.

**Comment by cameroncondry** [comment]
@Wymrite Thank you for taking the time to review these changes! I feel you two have more intimate knowledge of the game and KS since you are actively playing.

File: README.md [hunk 3]
```diff
@@ -19,69 +21,81 @@ You can also permanently install Kitten Scientists with a userscript manager.
...
+In this section, you can select which craftable resources you want to craft automatically.
 
-Furthermore, each resource can be set to be *limited* crafted. Limited crafting behaves normally when the resource exceeds the trigger value, but behaves differently when beneath the trigger value or for a resource that lacks a storage capacity. It crafts resources in an attempt to keep both the inputs and outputs in proportion. (Meaning if the player has 2000 minerals and 0 slabs, and each slab costs 50 minerals, it will craft 20 slabs for new totals of 1000 minerals and 20 slabs).
+When you enable the **Limited** option for a resource, then only a portion of the source materials is crafted into the desired resource. The portion is currently set to 50% (this is currently not configurable).
```
[the remark this answers is not in the document]
Comment (oliversalzburg) [reply README.md]: Good call. I went with a single line break initially, but a new paragraph looks cleaner. WIll be fixed.

File: README.md [hunk 4]
```diff
@@ -19,69 +21,81 @@ You can also permanently install Kitten Scientists with a userscript manager.
...
+This means that if you have 1000 beams, and you have crafting of scaffolds enabled, then only 500 beams would be crafted into scaffolds. This is especially useful for when you want to split up source materials between different craftable resources.
 
-### Upgrading
+#### Resources
 
-Techs and workshop upgrades are automatically bought when affordable, prioritizing the workshop. Buildings can be set to be automatically upgraded to their second stage as soon as upgrading would not significantly impact income. Space missions can automatically be conducted, and trading partners automatically discovered.
+By clicking the **Resources** label, you can access the fine-grained resource management.
 
-### Trading
+For each resource, you can set a *consumption rate* (60% by default) and a *stock* (0 by default). The consumption rate defines how much of the trigger value times the storage capacity that autocrafting is allowed to consume (meaning with the default consumption rate and a trigger value of 100%, autocrafting would leave 40% of the resource's storage capacity). The stock setting tells KS to ignore the specified amount of this resource.
```
[the remark this answers is not in the document]
Comment (oliversalzburg) [reply README.md]: I changed the last line to your suggestion.

File: README.md (hunk 1 above)
Comment (oliversalzburg) [reply README.md]: Yeah, I didn't like that either, but didn't know how to make it better at the time. I'll try to come up with something better.

File: README.md [hunk 5]
```diff
@@ -19,69 +21,81 @@ You can also permanently install Kitten Scientists with a userscript manager.
...
 
-Automatically build embassies.
+In this section, you can select which buildings from the **Time** page you want to have built automatically.
 
-Autofeed necrocorns to leviathans.
+### Options
 
-### Game Log
+- **Observe Astro Events**: Automatically observe astronomical events as they happen.
+- **Hold Festivals**: Automatically hold festivals as soon as the previous one ends and enough resources are available.
```
[the remark this answers is not in the document]
Comment (oliversalzburg) [reply README.md]: Fixed 👍

File: README.md (hunk 2 above)
Comment (oliversalzburg) [reply README.md]: Awesome. Fixed

**Comment by cameroncondry** [comment]
Excellent, looks great! I will go ahead and merge, but @Wymrite feel free to point out any issues and we can see about addressing them in another PR.

**Comment by oliversalzburg** [comment]
I also had a few minor changes I apparently forgot to push. I will take care of that later.

**Pull request merged**